src/main/java/org/prgrms/prgrmsspring/domain/Command.java

```diff
@@ -0,0 +1,35 @@
+package org.prgrms.prgrmsspring.domain;
+
+import org.prgrms.prgrmsspring.controller.ApplicationController;
+
+import java.util.function.Consumer;
+
+public enum Command {
+    EXIT("exit the program.", ApplicationController::exit),
+    CREATE("create a new voucher.", ApplicationController::create),
+    LIST("list all vouchers.", ApplicationController::list),
+    BLACKLIST("list all blacklist people", ApplicationController::showBlackList);
+
+    private final String document;
+    private final Consumer<ApplicationController> consumer;
+
+    Command(String document, Consumer<ApplicationController> consumer) {
+        this.document = document;
+        this.consumer = consumer;
+    }
+
+    public String getDocument() {
+        return document;
+    }
+
+    public static Command of(String name) throws IllegalArgumentException {
+        if (name.toLowerCase().equals(name)) {
+            return valueOf(name.toUpperCase());
+        }
+        return valueOf(name);
+    }
```

Comment:
```suggestion
    public static Command of(String name) throws IllegalArgumentException {
        return valueOf(name.toUpperCase());
    }
```

조건 처리가 필요한 경우가 어떤 상황에서 필요한 걸까요?

Reply:
소문자로 들어오는 경우 대문자로 변경하는 로직을 조건문에 추가한 것인데, 
말씀하신 것처럼 그냥 처음부터 대문자로 변경해서 하는 것이 무방해보입니다..!!